nixos/modules/services/web-servers/darkhttpd.nix

```diff
@@ -0,0 +1,77 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.darkhttpd;
+
+  optionsToArgs = concatStringsSep " " ([
+    cfg.rootDir
+    "--port ${toString cfg.port}"
+    "--addr ${cfg.address}"
+  ] ++ cfg.extraConfig
+    ++ optional cfg.hideServerId             "--no-server-id"
+    ++ optional config.networking.enableIPv6 "--ipv6");
+
+in {
+  options.services.darkhttpd = with types; {
+    enable = mkEnableOption "DarkHTTPd web server";
+
+    port = mkOption {
+      default = 80;
+      type = int;
```

Comment:
Can use `ints.u16` now

Reply:
Any particular reason other than shaving off 2 bytes?

Reply:
6 bytes.... :man_facepalming:

Reply:
It only allows valid port values, as opposed to `int`.

Reply:
:+1: